src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -181,4 +185,40 @@ public static HttpHost getBriefCaseProxyConnection() {
     }
     return null;
   }
+
+  /**
+   * Persist the user's decision to allow/disallow their behaviour being tracked.
+   * @param value (required) the boolean value representing the user's decision.
+   */
+  public static void setBriefcaseTrackingConsentProperty(Boolean value) {
+    if (value == null) {
+      Preference.APPLICATION_SCOPED.remove(BRIEFCASE_TRACKING_CONSENT_PROPERTY);
+    } else {
+      Preference.APPLICATION_SCOPED.put(BriefcasePreferences.BRIEFCASE_TRACKING_CONSENT_PROPERTY, value.toString());
+    }
+  }
+
+  /**
+   * Get the user's persisted decision regarding their consent to being tracked.
+   * @return the boolean representation of the user's consent to being tracked.
+   */
+  public static Boolean getBriefcaseTrackingConsentProperty() {
+    return Boolean.valueOf(
+            Preference.APPLICATION_SCOPED.get(BriefcasePreferences.BRIEFCASE_TRACKING_CONSENT_PROPERTY, Boolean.FALSE.toString())
+    );
+  }
+
+  /**
+   * Get the persisted UUID for the current user.
+   * <P>Note that the method will generate and persist a UUID if the user doesn't have one.</P>
+   * @return the String representation of the user's UUID.
+   */
+  public static String getUniqueUserID() {
+    String defaultUuidValue = "UUID missing, defaulting to this value";
```

Comment:
The important noun in this noun phrase is message, not value. It’s a message about a missing UUID.